geometry_msgs/Point start
geometry_msgs/Point end
---
nav_msgs/Path path